bin/resty

```diff
@@ -130,7 +130,15 @@ my $conns = $conns_num || 64;
 my @nameservers;
 
 if (!$resolve_ipv6) {
-    unshift @nameservers, "ipv6=off";
+    # no error check to let the later exec() fail.
+    # worst case: ipv6=off is not added to the nginx
+    # conf, but the worker won't spawn anyways...
+    my $stderr = `$nginx_path -V 2>&1 1>/dev/null`;
+    #warn $stderr;
+
+    if (index($stderr, "--with-ipv6") != -1) {
```

Comment:
@thibaultcha Will propose a patch for the nginx core to the `openresty/openresty` repo to silently ignore the `ipv6=off` directive argument when IPv6 is not enabled? (But we need an alert when the user specifies `ipv6=on` when IPv6 is not enabled.) We can have this patch in the openresty bundle's nginx core first anyway. It seems to be the easiest.